examples/ss3_template_files/starter.ss

```diff
@@ -1,44 +1,66 @@
-#V3.30.19.01;_safe;_compile_date:_Apr 15 2022;_Stock_Synthesis_by_Richard_Methot_(NOAA)_using_ADMB_12.3
+#V3.30.20.00;_safe;_compile_date:_Sep 30 2022;_Stock_Synthesis_by_Richard_Methot_(NOAA)_using_ADMB_13.0
 #_Stock_Synthesis_is_a_work_of_the_U.S._Government_and_is_not_subject_to_copyright_protection_in_the_United_States.
 #_Foreign_copyrights_may_apply._See_copyright.txt_for_more_information.
 #_User_support_available_at:NMFS.Stock.Synthesis@noaa.gov
 #_User_info_available_at:https://vlab.noaa.gov/group/stock-synthesis
 #_Source_code_at:_https://github.com/nmfs-stock-synthesis/stock-synthesis
 
-2021_model_species.dat
-2021_model_species.ctl
+#C starter comment here
+# PFMC: #############################################################
+# PFMC: .[a-z].ss ensures data and control file are first.
+# PFMC: #############################################################
+.data.ss
+.control.ss
 0 # 0=use init values in control file; 1=use ss.par
-1 # run display detail (0 = none, 1 = one brief line of display for each iterations, 2=fuller display per iteration)
+# PFMC: #############################################################
+# PFMC: Console and file output is verbose by default.
+# PFMC: echoinput.sso and parmtrace.sso are useful for debugging.
+# PFMC: Turn off output to run models faster.
+# PFMC: #############################################################
+2 # run display detail (0,1,2)
 1 # detailed output (0=minimal for data-limited, 1=high (w/ wtatage.ss_new), 2=brief, 3=custom) 
 # custom report options: -100 to start with minimal; -101 to start with all; -number to remove, +number to add, -999 to end
-0 # write 1st iteration details to echoinput.sso file (0,1) 
-0 # write parm values to ParmTrace.sso (0=no,1=good,active; 2=good,all; 3=every_iter,all_parms; 4=every,active)
-1 # write to cumreport.sso (0=no, 1=like&timeseries; 2=add survey fits)
-1 # Include prior_like for non-estimated parameters (0=only calc for active parameters, 1=calc for prior for all parameters with priors) 
-# Turn off prior_like if profiling over a parameter with a prior if you do not want to include the prior likelihood 
-1 # Use Soft Boundaries to aid convergence (0 = omit, 1 = use) (recommended)
+1 # write 1st iteration details to echoinput.sso file (0,1) 
+4 # write parm values to ParmTrace.sso (0=no,1=good,active; 2=good,all; 3=every_iter,all_parms; 4=every,active)
+2 # write to cumreport.sso (0=no,1=like&timeseries; 2=add survey fits)
```

Comment:
`cumreport.sso` doesn't get deleted each time the model runs, so can be used to accumulate information across model runs. This is an alternative to either using separate directories for different model runs or appending a run number to the Report.sso filename (e.g. by `r4ss::jitter()`). I'm not aware of anyone other than Rick using this file. 

The information in it is a subset of that provided by Report.sso file for each individual run but the file size could grow much larger over time if the user doesn't manually delete it. I would suggest setting this to 0 by default and allow the user to change it if they have a specific reason to do so.

Reply:
Resolving this because upon a search of the SS3 for `cumreport` I found all of the information 😜